manifests/0000_05_config-operator_01_scheduling.crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: scheduling.config.openshift.io
```

Comment:
Why isn't your resource plural like all the others?

Reply:
`schedulings`?